backend/src/test/java/trainingjournal/backend/controller/PlanControllerTest.java

```diff
@@ -0,0 +1,61 @@
+package trainingjournal.backend.controller;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.web.servlet.MockMvc;
+import trainingjournal.backend.repository.ExerciseRepository;
+import trainingjournal.backend.repository.PlanRepository;
+import trainingjournal.backend.service.PlanService;
+
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.csrf;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureMockMvc
+class PlanControllerTest {
+
+    @Autowired
+    MockMvc mockMvc;
+
+    @Autowired
+    ExerciseRepository exerciseRepository;
+    @Autowired
+    PlanRepository planRepository;
+
+    @WithMockUser(username="StandardUser")
+    @Test
+    void test_getAllExercises() throws Exception{
+        mockMvc.perform(get("/api/exercises"))
+                .andExpect(content().json("""
+[]
+"""));
+    }
+
+    @DirtiesContext
+    @WithMockUser(username="StandardUser")
+    @Test
+    void test_addNewExercise() throws Exception{
+
+        mockMvc.perform(post("/api/exercises/")
+                        .content("newExercise")
+                        .with(csrf())
+                       )
+
+                .andExpect(content().json("""
+{
+"description" : "newExercise",
+"repeats" : 0,
+"sets" : 0,
+"weight" : 0
```

Comment:
Hier fällt mir gerade etwas auf das ich merkwürdig finde
Wenn der User eine Übungsdefinition anlegt, hat er
- die Übung und
- die Anzahl der Sätze etc

aber, wenn man im PlanController eine Übung zum Katalog aller bekannten Übungen hinzufügt, fügt man ja eigentlich nur einen Namen in Form eines Strings hinzu. D.h. im Gesamtkatalog braucht man doch eigentlich gar ncht alle anderen Informationen? Zumindest ist es, wie ich sehe, der aktuelle Stand des Projekts